habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/migration/RemoveMonorepoGroupMigration.java

```diff
@@ -26,45 +26,40 @@
  * project's README.md, this prevents these dependencies from causing issues when Poetry projects are exported in
  * development releases.
  */
-public class CustomMonorepoGroupMigration extends AbstractMigration {
+public class RemoveMonorepoGroupMigration extends AbstractMigration {
 
-    public static final Logger logger = LoggerFactory.getLogger(CustomMonorepoGroupMigration.class);
+    public static final Logger logger = LoggerFactory.getLogger(RemoveMonorepoGroupMigration.class);
 
     protected Map<String, TomlReplacementTuple> replacements = new HashMap<>();
 
-    protected boolean hasExistingMonoRepoDependenciesGroup;
+    private boolean hasExistingDependenciesGroup = false;
 
     @Override
     protected boolean shouldExecuteOnFile(File file) {
         replacements.clear();
-        hasExistingMonoRepoDependenciesGroup = false;
-
         boolean shouldExecute = false;
         try (FileConfig tomlFileConfig = FileConfig.of(file)) {
             tomlFileConfig.load();
 
-            Optional<Config> toolPoetryDependencies = tomlFileConfig.getOptional(TomlUtils.TOOL_POETRY_DEPENDENCIES);
-            if (toolPoetryDependencies.isPresent()) {
-                Config foundDependencies = toolPoetryDependencies.get();
+            Optional<Config> toolPoetryMonorepoDependencies = tomlFileConfig.getOptional(TomlUtils.TOOL_POETRY_GROUP_MONOREPO_DEPENDENCIES);
+            if (toolPoetryMonorepoDependencies.isPresent()) {
+                Config foundDependencies = toolPoetryMonorepoDependencies.get();
                 Map<String, Object> dependencyMap = foundDependencies.valueMap();
 
                 for (Map.Entry<String, Object> dependency : dependencyMap.entrySet()) {
                     String packageName = dependency.getKey();
                     Object packageRhs = dependency.getValue();
                     if (TomlUtils.representsLocalDevelopmentVersion(packageRhs)) {
```

Comment:
S: Since we're removing the monorepo group entirely, I don't think we care to check whether it's a path dependency here -- we just want to move _all_ dependencies in that group. (In practice, it should only be path deps)